addons/block_code/simple_nodes/simple_character/simple_character.gd

```diff
@@ -83,24 +104,44 @@ func _ready():
 
 func simple_setup():
 	add_to_group("affected_by_gravity", true)
+	_setup_actions()
 	_texture_updated()
 
 
-func get_custom_class():
-	return "SimpleCharacter"
+func _setup_actions():
+	if(InputMap.has_action("player_1_left")):
+		return
```

Comment:
I was concerned that this would run in the editor (not just when the game is launched) and edit the project's input map. That would be problematic because it would mean that subsequent changes here would not be applied to the game. But in fact this works as desired: it is not run in the editor. (I don't actually know why because this is a `@tool`...)